homework/index.js

```diff
@@ -30,17 +35,122 @@
     return elem;
   }
 
+  function addRow(tBody, label, value) {
+    const row = createAndAppend('tr', tBody);
+    if (value === null) {
+      createAndAppend('td', row, { text: `${label} :`, class: 'label' });
+      createAndAppend('td', row, {
+        text: `No ${label.toLowerCase()} is provided for this repository.`,
+      });
+    } else {
+      createAndAppend('td', row, { text: `${label} :`, class: 'label' });
+      createAndAppend('td', row, { text: value });
+    }
+    return row;
+  }
```

Comment:
This function can indeed be made simpler. As a first approach, you could move the common part of the `if` and `else` clause out of the `if` statement:

```js
function addRow(tBody, label, value) {
  const row = createAndAppend('tr', tBody);
  createAndAppend('td', row, { text: `${label} :`, class: 'label' });
  if (value === null) {
    createAndAppend('td', row, {
      text: `No ${label.toLowerCase()} is provided for this repository.`,
    });
  } else {
    createAndAppend('td', row, { text: value });
  }
  return row;
}
```

A further  improvement is to create an intermediate variable and use a ternary operator to check for `null`:

```js
function addRow(tBody, label, value) {
  const text =
    value != null ? value : `No ${label.toLowerCase()} is provided for this repository.`;
  const row = createAndAppend('tr', tBody);
  createAndAppend('td', row, { text: `${label} :`, class: 'label' });
  createAndAppend('td', row, { text });
  return row;
}
```

Notes:
1. When comparing to `null` you can use the non-strict operators `==` and `!=`. In this way, `null` and `undefined` will both meet the comparison condition. This is the only situation where it is recommended to use the non-strict versions.
2. Note the syntax `{ text }`. This is ES6 shorthand notation for `{ text: text }`.